css/src/components/avatarSelection.css

```diff
@@ -0,0 +1,110 @@
+/* Selection avatar group */
+.SelectionAvatarGroup {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+}
+
+/* Selection Avatar List */
+
+.SelectionAvatarGroup-item {
+  margin-right: calc(var(--spacing-m) * -1);
+  border-radius: 50%;
+  display: flex;
+  transition: margin-top var(--duration--fast-01) var(--standard-productive-curve);
+}
+
+.SelectionAvatarGroup-item:focus,
+.SelectionAvatarGroup-item:focus-visible {
+  outline: 3px solid var(--primary-shadow);
+  outline-offset: 3px;
+}
+
+.SelectionAvatarGroup-item:active {
+  outline: var(--spacing-s) solid #00509f;
+  outline-offset: var(--spacing-xs);
+}
+
+.SelectionAvatarGroup-item:hover {
+  margin-top: calc(-1 * var(--spacing));
```

Comment:
Please move the children element as moving the same element on hover will cause fluctuation. 
1. mouse entered (hover ) : margin applied
2. mouse out ( as element moved ) : margin removed
These two will repeat in loop when the mouse is in outer edge ( calc(-1 * var(--spacing)); ). 
Please look into this as it is happening on mouse hover.